### DISCUSSION TITLE
에셋 구입 건 관련

### DISCUSSION THREAD

**Discussion by fkdl0048** [The-Day-In-1984/1984_Project#49] [Ideas]
에셋 저희팀 뭘 사면 좋을 지 생각을 해봤는데..

아래 에셋 쉐이더를 잘 사용하면 좀 좋을 것 같은..?

https://assetstore.unity.com/packages/vfx/shaders/all-in-1-sprite-shader-156513

**Comment by ccmuk58** [comment]
기획이 좀 더 나와 봐야 알겠지만 , , 

탐나긴 하네요 ,,,,, ○0○

**Comment by kyoonge** [comment]
오 먼지로 사라지는거 좋아보여요
